crates/utilities/cli/src/log.rs

```diff
@@ -1,40 +1,157 @@
 //! Arguments for logging.
 
-use clap::{ArgAction, Args};
-use tracing_subscriber::EnvFilter;
+use std::path::PathBuf;
 
-use crate::{init_tracing_subscriber, tracing::LogFormat};
+use clap::{ArgAction, Args, ValueEnum};
+use serde::{Deserialize, Serialize};
+use tracing::level_filters::LevelFilter;
+
+use crate::tracing::LogFormat;
 
 /// Global configuration arguments.
-#[derive(Args, Debug, Default, Clone)]
+#[derive(Args, Debug, Default, Serialize, Deserialize, Clone)]
 pub struct LogArgs {
-    /// Verbosity level (0-5).
-    /// If set to 0, no logs are printed.
+    /// Verbosity level (1-5).
     /// By default, the verbosity level is set to 3 (info level).
+    ///
+    /// This verbosity level is shared by both stdout and file logging (if enabled).
     #[arg(
-        long,
-        short,
+        long = "logs.level",
```

Comment:
What's the purpose of `logs.level`? It's an `ArgAction::Count` var; Think we should likely keep it as just `v`. With the long variant, you can't do `--logs.level=3` because of the `action`, it'd be `--logs.levellogs.levellogs.level`.